docs/integrations/guides.mdx

```diff
@@ -16,6 +16,12 @@ icon: 'code'
 
 Check out the [Next.js + Vercel example](https://github.com/browserbase/stagehand-nextjs-quickstart) to see how to build a Next.js app and one-click deploy it to Vercel.
```

Comment:
@miguelg719 @Kylejeong2 this MDX file is excluded from the nav - Is this on purpose?